ugvc/joint/compress_gvcf.py

```diff
@@ -0,0 +1,193 @@
+import pysam
```

Comment:
Can you please add documentation for the script + type annotations

Reply:
Can you explain what does miss in the documentation?